skipper.go

```diff
@@ -46,6 +46,9 @@ const DefaultPluginDir = "./plugins"
 // Options to start skipper.
 type Options struct {
 
+	//WhitelistedHealthcheckCIDR appends the whitelisted IP Range to the inernalIPS range for healthcheck purposes
```

Comment:
Please add a whitesapce before comment and first word:

    // WhitelistedHealthcheckCID